docs/pageobjects.md

```diff
@@ -3,15 +3,14 @@
 UI of your web application has interaction areas which can be shared across different tests.
 To avoid code duplication you can put common locators and methods into one place.
 
-## PageObjects
+## PageObject Generation
```

Comment:
Gotcha! My intention was to remove the duplicate headers which makes hyperlink generation much cleaner - hopefully my update will be the best of both worlds! :)